crucible-llvm/src/Lang/Crucible/LLVM/MemModel/Strings.hs

```diff
@@ -0,0 +1,52 @@
+-- TODO(TODO: file an issue, put number here): Move the definitions of the
```

Comment:
Reviewers: I will file this issue and update this TODO if you like the idea of moving the string-related functions into this new module.